src/Component/CompositionContext/CompositionContext.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+export interface Compositions {
+  LineEditor?: {
+    widthField?: React.ReactElement | false;
+    colorField?: React.ReactElement | false;
+    opacityField?: React.ReactElement | false;
+    dashPatternField?: React.ReactElement | false;
+    dashOffsetField?: React.ReactElement | false;
+    capField?: React.ReactElement | false;
+    JoinField?: React.ReactElement | false;
+    GraphicStrokeField?: React.ReactElement | false;
+    GraphicFillField?: React.ReactElement | false;
+  }
+}
+
+export const CompositionContext = React.createContext({});
```

Comment:
missing newline

Reply:
solved in https://github.com/terrestris/geostyler/pull/1186/commits/4fcbbd578e14eb8c7466107a521dd20ed8a860f5